assets/JS/temp-password.js

```diff
@@ -0,0 +1,30 @@
+
+var counter = 0;
+function randomPassword(){
```

Comment:
Read Ejs docs about passing data to views. Don't generate password at client side.